bool request
---
bool success
uint32 wp_transferred